source/includes/gcp/_load_balancer.md

```diff
@@ -48,18 +48,101 @@ Retrieve a list of all load balancers in a given [environment](#administration-e
 
 Attributes | &nbsp;
 ------- | -----------
-`backends`<br/>*List<String>* | The list of backends for this resource
-`backendService`<br/>*String* | Server-defined URL for corresponding backend service
-`forwardingRule`<br/>*String* | Server-defined URL for corresponding forwarding rule
-`id`<br/>*String* | The id of this resource, which should be the same id as corresponding forwarding rule
+`backendServices`<br/>*List<String>* | The list of backends for this resource
+`forwardingRules`<br/>*List<String>* | The list of forwarding rules for this resource
+`id`<br/>*String* | The id of this resource, which should be the same id as corresponding Url Map
 `name`<br/>*String* | The name of this resource
-`protocol`<br/>*String* | The protocol of this resource
-`publicAddress`<br/>*String* | The public address which allow user access their service quickly
-`shortBackends`<br/>*String* | The list of backends for this resource in string format 
-`targetProxy`<br/>*String* | Server-defined URL for corresponding target proxy
+`protocol`<br/>*String* | The protocol of this resource. If there are multiple target proxies, and at least one of them is HTTPS, the protocol for the load balancer is set to HTTPS. Valid values are HTTP or HTTPS.
+`publicAddress`<br/>*List<String>* | The list of external addresses
```

Comment:
maybe plural since it's multiple addresses?

Reply:
renamed to `publicAddresses`